sonic_data_client/db_client.go

```diff
@@ -39,6 +39,8 @@ type Client interface {
 	// The service will stop upon detection of poll channel closing.
 	// It should run as a go routine
 	PollRun(q *queue.PriorityQueue, poll chan struct{}, w *sync.WaitGroup, subscribe *gnmipb.SubscriptionList)
+	// Poll to service AppDB only
+	AppDBPollRun(q *queue.PriorityQueue, poll chan struct{}, w *sync.WaitGroup, subscribe *gnmipb.SubscriptionList)
```

Comment:
>AppDBPollRun [](http://example.com/codeflow?start=1&length=12)

I could not find the code calling it, do you miss some file?

Reply:
Fixed